pathsimanalysis/__init__.py

```diff
@@ -10,13 +10,10 @@
 
 due.cite(Doi("10.1371/journal.pcbi.1004568"),
```

Comment:
We're importing duecredit directly from upstream mdanalysis - do we care about this? I don't actually know if it matters and/or if it might be better not to do this. The alternative is just to add the vendored file here (it's a reasonably cost-free process).

Reply:
One still needs to install duecredit IIRC and I'd assume that people would install it together with MDA if they cared about it. So I think it's ok to leave it imported from MDA.

At the end of the day, this package does not run without MDA so we can use it from there.